### PR TITLE
Remove some dead code in notifyexternals

### PR DESCRIPTION
**Pull request by adityamandaleeka**
This change removes a bunch of dead code related to `ApartmentTearDownHandler`.

@jkotas PTAL

### CODE REVIEW

**Comment by jkotas** [comment]
LGTM

File: src/vm/notifyexternals.cpp [hunk 1]
```diff
@@ -14,228 +14,10 @@
...
-#include "ctxtcall.h"
 #include "notifyexternals.h"
 #include "mdaassistants.h"
 
-DEFINE_GUID(CLSID_ComApartmentState, 0x00000349, 0, 0, 0xC0,0,0,0,0,0,0,0x46);
-static const GUID IID_ITeardownNotification = { 0xa85e0fb6, 0x8bf4, 0x4614, { 0xb1, 0x64, 0x7b, 0x43, 0xef, 0x43, 0xf5, 0xbe } };
-static const GUID IID_IComApartmentState = { 0x7e220139, 0x8dde, 0x47ef, { 0xb1, 0x81, 0x08, 0xbe, 0x60, 0x3e, 0xfd, 0x75 } };
-
-static IComApartmentState* g_pApartmentState = NULL;
 static ULONG_PTR      g_TDCookie = 0;
```
Comment (jkotas) [inline src/vm/notifyexternals.cpp]: Delete `g_TDCookie`? It was only used in the code that you are deleting.

**Comment by adityamandaleeka** [comment]
@jkotas Thanks, I deleted that too.

**Comment by adityamandaleeka** [comment]
@dotnet-bot test this please

**Pull request merged**